pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -2572,6 +2578,11 @@ self: super: {
     lsp = self.lsp_1_5_0_0;
   };
 
+  vector-hashtables = unmarkBroken (super.vector-hashtables.override {
```

Comment:
The `unmarkBroken` is unnecessary after adapting `configuration-hackage2nix/broken.yaml`.